### PR TITLE
Proposed Therapeutics Manuscript Content/Structure

### PR DESCRIPTION
**Pull request by rando2**
This builds on the original therapeutics section (minus nutraceuticals, #734) and the text from the introduction and discussion that is pertinent to this section (#735). ~It is dependent on #735 and can't be merged until that one is (and the changes here will become a bit simpler once that one is merged).~ (All set now, thanks @ajlee21!)

Please see #738 for additional steps that need to happen (e.g., updating vaccine info as it changes)

### Description of the proposed additions or changes

This should now be a standalone manuscript. Feedback to improve it is very welcome, though!

Current word count: 17,905 (including headers, figure captions, citations, md for inserting figures, comments, etc.)

### Related issues

See above

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/60.methods.md [hunk 1]
```diff
@@ -4,6 +4,19 @@
...
+While having information available is valuable to efforts to understand and combat COVID-19, many contributions come from researchers across a wide range of fields who have varying degrees of experience working on coronaviruses and related topics.
+The volume of information available, much of which has not gone through rigorous peer review, presents a significant challenge to individual efforts to keep abreast of the state of COVID-19 research [@doi:10.1038/s42254-020-0175-7].
+However, research on these topics is proceeding so rapidly that any static review is likely to quickly become dated.
+Our goal as a community is to consolidate information about the virus in the context of related viruses and to synthesize rapidly emerging literature centered on the diagnosis and treatment of COVID-19.
+We used an open publishing framework, Manubot [@doi:10.1371/journal.pcbi.1007128], to manage hundreds of contributions from the community to create a living, scholarly document.
+We designed software to generate figures, such as Figure @fig:csse-deaths, that automatically update using external data sources.
```
Comment (ajlee21) [inline content/60.methods.md:14]: Is there code for this somewhere? Does it make sense to link to it here/add it to this repo?

**Review by ajlee21 — APPROVED** [review]
LGTM, just one minor comment

File: content/60.methods.md (hunk 1 above)
Comment (rando2) [reply content/60.methods.md]: @ajlee21 It definitely should! That manuscript needs some major work. I'm tagging @mprobson who has said he'd like to put some work into making it read more like a computer/data science education paper this week!

**Comment by rando2** [comment]
@ajlee21 approved and this has been open for a while, so I'm going to merge! Thanks so much @ajlee21!!!

File: content/60.methods.md (hunk 1 above)
Comment (agitter) [reply content/60.methods.md]: @ajlee21 this figure generation code is in this repo but on a separate branch.  You can find it in the subdirectories here https://github.com/greenelab/covid19-review/tree/external-resources

Linking to it directly once we start working on this standalone manuscript is a good idea.  I also tried archiving the entire repository with Software Heritage in the fall, but it failed.  We'll want a backup plan for archiving this code.

**Pull request merged**